components/playlist/browser/resources/components/player.tsx

```diff
@@ -103,14 +119,30 @@ export default function Player () {
         onPlay={() => getPlayerActions().playerStartedPlayingItem(currentItem)}
         onPause={() => getPlayerActions().playerStoppedPlayingItem(currentItem)}
         onEnded={() => {
+          if (currentItem) {
+            notifyEventsToTopFrame({
+              type: PlaylistTypes.PLAYLIST_LAST_PLAYED_POSITION_OF_CURRENT_ITEM_CHANGED,
+              data: {
+                ...currentItem,
+                lastPlayedPosition: videoElement!.duration
+              }
+            })
+          }
+
           getPlayerActions().playerStoppedPlayingItem(currentItem)
           getPlayerActions().playNextItem() // In case the current item is the last one, nothing will happen
         }}
-        src={
-          videoElement?.src === currentItem?.mediaSource.url
-            ? currentItem?.mediaSource.url // We were already playing the item with the original source url instead of cached file
-            : currentItem?.mediaPath.url
-        }
+        onTimeUpdate={() => {
+          if (!currentItem) return
+
+          notifyEventsToTopFrame({
```

Comment:
Do we need to debounce this? (if it only fires once per second, it's probably fine)

Reply:
Yes, it's filed per sec. When I was implementing this, I was wondering what frequency would be "too frequent". Is once per sec usually okay?

Reply:
It doesn't feel too excessive - I guess if it cause problems we could change to every 5 or every 10. Probably fine to revisit later :smile: